package.json

```diff
@@ -23,19 +23,20 @@
     "url": "https://github.com/pwlmaciejewski/imghash/issues"
   },
   "engines": {
-    "node": ">=4.x.x"
+    "node": ">=8.x.x"
   },
   "homepage": "https://github.com/pwlmaciejewski/imghash#readme",
   "devDependencies": {
     "chai": "^3.5.0",
     "hamming-distance": "^1.0.0",
-    "mocha": "^2.4.5"
+    "mocha": "^2.4.5",
+    "png-js": "^0.1.1"
```

Comment:
This `png-js` dev dependency was added by accident, right?

Reply:
Not an accident exactly... it seems that since blockhash still depends on png-js we need that dev dependency for the tests. I get this when running the tests if I leave it out:
```
> 07:59:28 fluxcapacitor:~/projects/imghash$ npm run test

> imghash@0.0.3 test /Users/ray/projects/imghash
> mocha

internal/modules/cjs/loader.js:797
    throw err;
    ^

Error: Cannot find module 'png-js'
Require stack:
- /Users/ray/projects/imghash/node_modules/blockhash/index.js
- /Users/ray/projects/imghash/index.js
- /Users/ray/projects/imghash/test/test.js
- /Users/ray/projects/imghash/node_modules/mocha/lib/mocha.js
- /Users/ray/projects/imghash/node_modules/mocha/index.js
- /Users/ray/projects/imghash/node_modules/mocha/bin/_mocha
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:794:15)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Module.require (internal/modules/cjs/loader.js:849:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/Users/ray/projects/imghash/node_modules/blockhash/index.js:7:11)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:849:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/Users/ray/projects/imghash/index.js:8:19)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:849:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/Users/ray/projects/imghash/test/test.js:2:17)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:849:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at /Users/ray/projects/imghash/node_modules/mocha/lib/mocha.js:220:27
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (/Users/ray/projects/imghash/node_modules/mocha/lib/mocha.js:217:14)
    at Mocha.run (/Users/ray/projects/imghash/node_modules/mocha/lib/mocha.js:469:10)
    at Object.<anonymous> (/Users/ray/projects/imghash/node_modules/mocha/bin/_mocha:404:18)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1025:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/ray/projects/imghash/node_modules/blockhash/index.js',
    '/Users/ray/projects/imghash/index.js',
    '/Users/ray/projects/imghash/test/test.js',
    '/Users/ray/projects/imghash/node_modules/mocha/lib/mocha.js',
    '/Users/ray/projects/imghash/node_modules/mocha/index.js',
    '/Users/ray/projects/imghash/node_modules/mocha/bin/_mocha'
  ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! imghash@0.0.3 test: `mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the imghash@0.0.3 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ray/.npm/_logs/2019-10-28T15_08_42_518Z-debug.log
```

Reply:
That's not perfect but let's leave it like that for now. 

- [ ] @pwlmaciejewski create an issue to replace `humming-distance` and remove `png-js` from dev deps